src/modules/logicModule/LM_PC_RecurringPayments_v1.sol

```diff
@@ -137,6 +138,16 @@ contract LM_PC_RecurringPayments_v1 is
         }
 
         emit EpochLengthSet(newEpochLength);
+
+        // Set the flags for the PaymentOrders
+        uint8 numOfFlags = 3; // The Module will use 3 flags
```

Comment:
Same for the setting of flags
Is this the way we go forward or do we want to enforce this via an abstract function?